packages/js/src/plugins/auctionHouseModule/models/Listing.ts

```diff
@@ -30,8 +30,8 @@ export type Listing = Readonly<{
   /** A model of the Auction House related to this listing. */
   auctionHouse: AuctionHouse;
 
-  /** The Nft or Sft with the associated token account. */
-  asset: NftWithToken | SftWithToken;
+  /** The Nft or Sft related to this listing. */
+  asset: Sft | Nft | NftWithToken | SftWithToken;
```

Comment:
Why? Listings have to have a token account as there's no such thing as a Public Listing. We're losing valuable information here.

Reply:
I wanted to fix the problem with missing token in direct sell due to public bid and skip the token loading